staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -43,11 +43,14 @@ import (
 	apitesting "k8s.io/apimachinery/pkg/api/testing"
 	metainternalversion "k8s.io/apimachinery/pkg/apis/meta/internalversion"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```

Comment:
@sttts serialization changes start from here down.  They look ok to me, can you double check them?